src/libmongoc/tests/test-mongoc-collection.c

```diff
@@ -6350,6 +6350,63 @@ test_remove_multi (void)
    mongoc_client_destroy (client);
 }
 
+static void
+test_timeout_ms (void)
+{
+   mongoc_client_t *client = mongoc_client_new ("mongodb://host/db_name");
+   mongoc_database_t *db;
+   mongoc_collection_t *coll = mongoc_client_get_collection (client, "db", "test");
+   mongoc_collection_t *coll2;
```

Comment:
To be safe, we should set `db` and `coll2` pointers to `NULL`. Here and for other new declarations.

Reply:
It should be fine to leave these newly-declared pointers undefined at the start of a test method.  It's our typical style to do so throughout the codebase.  I agree that it's safer to explicitly set to `NULL`, but given the current conventions in the codebase, I do feel it's a bit unnecessary to do so here (unless there is a push to change this convention that I'm not aware of).

Reply:
If you believe it's safer to do so, then we probably should. No need to leave the door open for easily preventable bugs because we've done so in the past. It'd be difficult to improve code quality - in this case, make our codebase safer - if we appeal to tradition.

That being said, I've certainly forgotten to do this at times, so feel free to point it out in my PRs as well.

Reply:
The reason I'm pushing back a bit here is that, to me, this kind of feedback falls into a category where, though it's not _purely_ a matter of personal preference, it's close, it isn't super substantive either, and it delays the review for little to no functional difference or benefit in the code.  The code as I have written it here is not incorrect, nor is it risky, since it is in a test.  I don't think that this sort of change is one we need to be pushing for.

I think there has to be a balance in our reviews between the ideal of following a golden standard of style, and following existing conventions in the codebase in a way that allows us to get code into production efficiently.

Perhaps @kevinAlbs has additional thoughts.

Reply:
To weigh in, I do agree. Raising codebase quality is an admirable goal. But unless we have an agreed team standard, style or non-correctness changes may not be consistent and do add to review turnaround time. I think there is a good middle ground though. Code reviews do provide a good forum for sharing ideas and learning as well. I am happy to discuss offline.